File: ios/Classes/data/RelationshipType.swift
//
//  RelationshipType.swift
//  contacts
//
//  Created by Maurits van Beusekom on 28/08/2018.
//

import Foundation

enum RelationshipType : String, Codable {
    case father = "father"
    case mother = "mother"
    case parent = "parent"
    case brother = "brother"
    case sister = "sister"
    case child = "child"
    case friend = "friend"
    case spouse = "spouse"
    case partner = "partner"
    case assistant = "assistant"
    case manager = "manager"
    case other = "other"
}

File: android/src/main/kotlin/com/baseflow/contactsplugin/data/RelationshipType.kt
package com.baseflow.contactsplugin.data

import com.google.gson.annotations.SerializedName

enum class RelationshipType {
    @SerializedName("father")
    FATHER,
    @SerializedName("mother")
    MOTHER,
    @SerializedName("parent")
    PARENT,
    @SerializedName("brother")
    BROTHER,
    @SerializedName("sister")
    SISTER,
    @SerializedName("child")
    CHILD,
    @SerializedName("friend")
    FRIEND,
    @SerializedName("spouse")
    SPOUSE,
    @SerializedName("partner")
    PARTNER,
    @SerializedName("assistant")
    ASSISTANT,
    @SerializedName("manager")
    MANAGER,
    @SerializedName("other")
    OTHER,
}